draft-amsuess-core-shopinc.md

```diff
@@ -209,6 +222,24 @@ Entry fields are:
 
   This field may be empty if the document describes that the option needs to be repeated when using this first value.
 
+* Subsequent Options
+
+This column defines how subsequent Uri-Path-Short options are to be initerpreted.
+There are four possible values for the entry:
+
+chained:
+: Subsequent options are to be looked up in this registry, and treated as an initial option.
```

Comment:
Chaining directly into this registry doesn't make sense for many values (`/.well-known/brski/.well-known/edhoc`?), and also doesn't need the CBOR escape hatch of not having the MSB set. I'd rather have those documents specify their own list, either hardcoded in their specification or using a registry mechanism available to them.